CHANGELOG.md

```diff
@@ -1,3 +1,82 @@
+# v1.11.0 (2022-11-15)
+
+## OS Changes
+
+* Prevent a panic early-boot-config when there is no IMDS Region ([#2493])
+* Update grub to 2.06-42 ([#2503])
+* Allow interface configuration via Ethernet permanent address ([#2519])
+* Allow bootstrap containers to manage swap ([#2537])
+* Add systemd analyze commands to troubleshooting log collection tool ([#2550])
+* Allow bootstrap containers to manage network configuration ([#2558])
+* Serialize bootconfig values correctly when the value is empty ([#2565])
+
+## Orchestrator Changes
+
+### ECS
+
+* Add additional ECS API configurations ([#2527])
+
+### Kubernetes
+
+* Update EKS-D to 1.23-6 for k8s-1.23 variants ([#2488])
+* Update EKS-D to 1.22-11 for k8s-1.22 variants ([#2490])
+* Add a timeout when calling EKS for configuration values ([#2566])
+* Enable IAM Roles Anywhere with the k8s `ecr-credential-provider plugin ([#2377], [#2553])
+
+## Platform Changes
+
+### AWS
+
+* Add driver support for AWS variants in hybrid environments ([#2554])
+
+## Build Changes
+
+* Add support for publishing to AWS organizations ([#2484])
+* Remove unnecessary dependencies when building grub ([#2495])
+* Switch to the latest Dockerfile frontend for builds ([#2496])
+* Prepare foundations for Secure Boot and image resigning ([#2505])
```

Comment:
Nit: I think "re-signing" might be more correct here

Reply:
lol, I'm glad it doesn't mean the PR author is resigning.